ProcessesApi/V1/Domain/SoleToJoint/SoleToJointConstants.cs

```diff
@@ -20,8 +21,9 @@ public static class SoleToJointStates
 
     public static class SoleToJointPermittedTriggers
     {
-        public const string CheckAutomatedEligibility = "CheckAutomatedEligibility";
         public const string CancelProcess = "CancelProcess";
+        public const string CloseProcess = "CloseProcess";
```

Comment:
Can this trigger be a part of SharedProcessConstants?

Reply:
Not really.. the issue is with the permittedTriggers list, if it's in the shared constants class it won't show up in the permittedTriggers array.